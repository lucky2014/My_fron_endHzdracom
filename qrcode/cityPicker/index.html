<!DOCTYPE html>
<!-- saved from url=(0089)file:///C:/Users/Administrator.LQ-20160307MDZP/AppData/Local/Temp/MarkdownPadPreview.html -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>城市选择</title>

<style type="text/css">
/* GitHub stylesheet for MarkdownPad (http://markdownpad.com) */
/* Author: Nicolas Hery - http://nicolashery.com */
/* Version: b13fe65ca28d2e568c6ed5d7f06581183df8f2ff */
/* Source: https://github.com/nicolahery/markdownpad-github */

/* RESET
=============================================================================*/

html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video {
  margin: 0;
  padding: 0;
  border: 0;
}

/* BODY
=============================================================================*/

body {
  font-family: Helvetica, arial, freesans, clean, sans-serif;
  font-size: 14px;
  line-height: 1.6;
  color: #333;
  background-color: #fff;
  padding: 20px;
  max-width: 960px;
  margin: 0 auto;
}

body>*:first-child {
  margin-top: 0 !important;
}

body>*:last-child {
  margin-bottom: 0 !important;
}

/* BLOCKS
=============================================================================*/

p, blockquote, ul, ol, dl, table, pre {
  margin: 15px 0;
}

/* HEADERS
=============================================================================*/

h1, h2, h3, h4, h5, h6 {
  margin: 20px 0 10px;
  padding: 0;
  font-weight: bold;
  -webkit-font-smoothing: antialiased;
}

h1 tt, h1 code, h2 tt, h2 code, h3 tt, h3 code, h4 tt, h4 code, h5 tt, h5 code, h6 tt, h6 code {
  font-size: inherit;
}

h1 {
  font-size: 28px;
  color: #000;
}

h2 {
  font-size: 24px;
  border-bottom: 1px solid #ccc;
  color: #000;
}

h3 {
  font-size: 18px;
}

h4 {
  font-size: 16px;
}

h5 {
  font-size: 14px;
}

h6 {
  color: #777;
  font-size: 14px;
}

body>h2:first-child, body>h1:first-child, body>h1:first-child+h2, body>h3:first-child, body>h4:first-child, body>h5:first-child, body>h6:first-child {
  margin-top: 0;
  padding-top: 0;
}

a:first-child h1, a:first-child h2, a:first-child h3, a:first-child h4, a:first-child h5, a:first-child h6 {
  margin-top: 0;
  padding-top: 0;
}

h1+p, h2+p, h3+p, h4+p, h5+p, h6+p {
  margin-top: 10px;
}

/* LINKS
=============================================================================*/

a {
  color: #4183C4;
  text-decoration: none;
}

a:hover {
  text-decoration: underline;
}

/* LISTS
=============================================================================*/

ul, ol {
  padding-left: 30px;
}

ul li > :first-child, 
ol li > :first-child, 
ul li ul:first-of-type, 
ol li ol:first-of-type, 
ul li ol:first-of-type, 
ol li ul:first-of-type {
  margin-top: 0px;
}

ul ul, ul ol, ol ol, ol ul {
  margin-bottom: 0;
}

dl {
  padding: 0;
}

dl dt {
  font-size: 14px;
  font-weight: bold;
  font-style: italic;
  padding: 0;
  margin: 15px 0 5px;
}

dl dt:first-child {
  padding: 0;
}

dl dt>:first-child {
  margin-top: 0px;
}

dl dt>:last-child {
  margin-bottom: 0px;
}

dl dd {
  margin: 0 0 15px;
  padding: 0 15px;
}

dl dd>:first-child {
  margin-top: 0px;
}

dl dd>:last-child {
  margin-bottom: 0px;
}

/* CODE
=============================================================================*/

pre, code, tt {
  font-size: 12px;
  font-family: Consolas, "Liberation Mono", Courier, monospace;
}

code, tt {
  margin: 0 0px;
  padding: 0px 0px;
  white-space: nowrap;
  border: 1px solid #eaeaea;
  background-color: #f8f8f8;
  border-radius: 3px;
}

pre>code {
  margin: 0;
  padding: 0;
  white-space: pre;
  border: none;
  background: transparent;
}

pre {
  background-color: #f8f8f8;
  border: 1px solid #ccc;
  font-size: 13px;
  line-height: 19px;
  overflow: auto;
  padding: 6px 10px;
  border-radius: 3px;
}

pre code, pre tt {
  background-color: transparent;
  border: none;
}

kbd {
    -moz-border-bottom-colors: none;
    -moz-border-left-colors: none;
    -moz-border-right-colors: none;
    -moz-border-top-colors: none;
    background-color: #DDDDDD;
    background-image: linear-gradient(#F1F1F1, #DDDDDD);
    background-repeat: repeat-x;
    border-color: #DDDDDD #CCCCCC #CCCCCC #DDDDDD;
    border-image: none;
    border-radius: 2px 2px 2px 2px;
    border-style: solid;
    border-width: 1px;
    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
    line-height: 10px;
    padding: 1px 4px;
}

/* QUOTES
=============================================================================*/

blockquote {
  border-left: 4px solid #DDD;
  padding: 0 15px;
  color: #777;
}

blockquote>:first-child {
  margin-top: 0px;
}

blockquote>:last-child {
  margin-bottom: 0px;
}

/* HORIZONTAL RULES
=============================================================================*/

hr {
  clear: both;
  margin: 15px 0;
  height: 0px;
  overflow: hidden;
  border: none;
  background: transparent;
  border-bottom: 4px solid #ddd;
  padding: 0;
}

/* TABLES
=============================================================================*/

table th {
  font-weight: bold;
}

table th, table td {
  border: 1px solid #ccc;
  padding: 6px 13px;
}

table tr {
  border-top: 1px solid #ccc;
  background-color: #fff;
}

table tr:nth-child(2n) {
  background-color: #f8f8f8;
}

/* IMAGES
=============================================================================*/

img {
  max-width: 100%
}
</style>
<!--<base href="file:\\\C:\Users\Administrator.LQ-20160307MDZP\Desktop\">--><base href=".">
</head>
<body>
<h2>城市选择插件使用说明</h2>
<ol>
<li>
<p>html</p>
<pre>&lt;script src="../js/jquery/jquery-1.8.3.min.js"&gt;
&lt;link type="text/css" rel="stylesheet" href="css/cityPicker.css" /&gt;
&lt;script src="js/cityPicker.js"&gt;
&lt;div id="cityPicker"&gt;
&lt;div&gt;
    &lt;p&gt;&lt;i data-num=""&gt;请选择省份&lt;/i&gt;&lt;em class="arrow"&gt;&lt;/em&gt;&lt;/p&gt;
    &lt;ul class="hide"&gt;&lt;/ul&gt;
&lt;/div&gt;
&lt;div&gt;
    &lt;p&gt;&lt;i data-num=""&gt;请选择城市&lt;/i&gt;&lt;em class="arrow"&gt;&lt;/em&gt;&lt;/p&gt;
    &lt;ul class="hide"&gt;&lt;/ul&gt;
&lt;/div&gt;
&lt;div&gt;
    &lt;p&gt;&lt;i data-num=""&gt;请选择县&lt;/i&gt;&lt;em class="arrow"&gt;&lt;/em&gt;&lt;/p&gt;
    &lt;ul class="hide"&gt;&lt;/ul&gt;
&lt;/div&gt;
&lt;/div&gt;</pre>
</li>
<li>
<p>js</p>
<pre>$(function(){
$("#cityPicker").cityPicker({
    url: "http://122.224.218.55:8121/liujia-enterprise-server/activity/getAreaInfo.do",
    hasCountry: true //是否显示县级
});
});</pre>
</li>
<li>
<p>css</p>
<pre>ul,li,p,dl,dt,dd,em{ margin: 0; padding: 0; list-style: none;}
.hide{ display: none;}
#cityPicker{ clear: both; margin: 20px;}
#cityPicker div{ display: inline-block; width: 160px; position: relative; margin-right: 10px;} 
#province{ margin-right: 10px;}
#cityPicker div&gt;p{ cursor: pointer; border: 1px solid #ccc; border-radius: 4px;
height: 28px; line-height: 28px; position: relative;}
#cityPicker div i{ display: block; width: 94%; position: absolute; 
font-style: normal; padding-left: 6%;}
#cityPicker div ul{ position: absolute; left: 0; top: 29px; border: 1px solid #ccc; 
border-radius: 4px; width: 158px; padding:6px 0;}
#cityPicker div li{ line-height: 28px; padding: 0 10px; cursor: pointer;}
#cityPicker div li:hover{ background: #06c; color: #fff;}
#cityPicker div em{display: inline-block; position: absolute; right: 10px;
top: 12px; width: 0; height: 0; margin-left: 2px; vertical-align: middle; 
border-top: 4px solid; border-right: 4px solid transparent; 
border-left: 4px solid transparent; }</pre>
</li>
<li>
<p><a href="http://122.224.96.245/index/cityPicker/demo/index.html">demo</a></p>
</li>
<li>
<p><a href="http://122.224.96.245/index/cityPicker/demo.rar">下载</a></p>
</li>
</ol>



<!-- This document was created with MarkdownPad, the Markdown editor for Windows (http://markdownpad.com) -->
</body></html>