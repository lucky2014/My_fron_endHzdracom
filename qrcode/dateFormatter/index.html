<!doctype HTML>
<html>
	<head>
		<meta charset="utf-8"/>
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=320,initial-scale=1.0,maximum-scale=1.0,user-scalable=0">
		<meta name="apple-mobile-web-app-capable" content="yes">
		<meta name="format-detection" content="telephone=no">
		<link rel="stylesheet" href="../sweetAlert/css/example.css">
		<title>将后台返回的日期字符串转化为标准日期格式</title>
	</head>
	<body>
		<a href="../index.html" class="back">返回</a>
		<h3>效果图</h3>
		<div class="examples" style="text-align:center;"><img src="img/1.png" /></div>

		<h3>代码(可以用查看源文件里找)</h3>
<pre>
formatDateFn: function(t, long) { //时间格式化，long参数是判断是否显示长日期（显示时间）格式
  var _this = this;
  t = new Date(t);
  var year = t.getFullYear();
  var month = _this.formatNumFn(t.getMonth() + 1);
  var date = _this.formatNumFn(t.getDate());
  var hour = _this.formatNumFn(t.getHours());
  var minute = _this.formatNumFn(t.getMinutes());
  var second = _this.formatNumFn(t.getSeconds());
  if (long == 'long') {
    return year + "-" + month + "-" + date + "   " + hour + ":" + minute + ":" + second;
  } else {
    return year + "-" + month + "-" + date;
  }
}
</pre>

	<h3></h3>
	</body>
</html>