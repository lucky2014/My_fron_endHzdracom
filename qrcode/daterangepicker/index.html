<!doctype HTML>
<html>
	<head>
		<meta charset="utf-8"/>
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=320,initial-scale=1.0,maximum-scale=1.0,user-scalable=0">
		<meta name="apple-mobile-web-app-capable" content="yes">
		<meta name="format-detection" content="telephone=no">
		<link rel="stylesheet" href="../js/jquery.ui/themes/base/jquery.ui.all.css" type="text/css">
		<link rel="stylesheet" href="../sweetAlert/css/example.css">
		<link rel="stylesheet" href="js/ui.daterangepicker.css" type="text/css" />

		<script src="../js/jquery/jquery-1.8.3.min.js"></script>
		<script src="../js/jquery.ui/jquery-ui-1.10.3.min.js"></script>
		<script src="js/moment.js"></script>
		<script src="js/daterangepicker.jQuery.js"></script>
		<script src="js/daterangepicker.js"></script>
  		
		<title>联级日期</title>
	</head>
	<body>
		<a href="../index.html" class="back">返回</a>
		<h3>效果图</h3>
		<div class="examples" style="text-align:center;"><img src="date.png" /></div>
		<h3>例子</h3>
		<ul class="examples">
			<li class="message">
				<div class="ui">
					<input type="text" class="input" id="datepicker" />
				</div>
<pre>initMyDates.dataPicker("#datepicker",0,function(){});</pre>
			</li>
		</ul>
		<h3></h3>

		<script>
		$(function(){
			initMyDates.dataPicker("#datepicker",0,function(){

			});
		});
		</script>
	</body>
</html>