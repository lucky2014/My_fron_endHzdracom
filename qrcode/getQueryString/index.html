<!doctype HTML>
<html>
	<head>
		<meta charset="utf-8"/>
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=320,initial-scale=1.0,maximum-scale=1.0,user-scalable=0">
		<meta name="apple-mobile-web-app-capable" content="yes">
		<meta name="format-detection" content="telephone=no">
		<link rel="stylesheet" href="../sweetAlert/css/example.css">
		<title>获取浏览器URL地址的某个参数值</title>
	</head>
	<body>
		<a href="../index.html" class="back">返回</a>
		<h3>效果图</h3>
		<div class="examples" style="text-align:center;"><img src="1.png" /></div>

		<h3>代码</h3>
		<ul class="examples">

			<li class="message">
			<div class="ui">
				<button>函数</button>
			</div>
<pre>
function getQueryString(name) { //获取URL的参数，isEit
	var reg = new RegExp('(^|&)' + name + '=([^&]*)(&|$)', 'i');
	var r = window.location.search.substr(1).match(reg);
	if (r != null) {
		return unescape(r[2]);
	}
	return null;
}
</pre>
			</li>

			<li class="message">
			<div class="ui">
				<button>使用方法</button>
			</div>
<pre>
	var string = getQueryString("templateId");
</pre>
			</li>
		</ul>

		<h3></h3>
	</body>
</html>