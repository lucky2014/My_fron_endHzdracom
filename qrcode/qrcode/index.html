<!doctype HTML>
<html>
	<head>
		<meta charset="utf-8"/>
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=320,initial-scale=1.0,maximum-scale=1.0,user-scalable=0">
		<meta name="apple-mobile-web-app-capable" content="yes">
		<meta name="format-detection" content="telephone=no">
		<link rel="stylesheet" href="../sweetAlert/css/example.css">
		<title>将URL网址转为二维码显示在页面上</title>
	</head>
	<body>
		<a href="../index.html" class="back">返回</a>
		<h3>效果图</h3>
		<div class="examples" style="text-align:center;"><img src="1.png" /></div>

		<h3>代码</h3>
		<ul class="examples">

			<li class="message">
			<div class="ui">
				<button>使用</button>
			</div>
<pre>
1、引用JS库后使用下面代码
$(".qrcode").html("");
	//查看示例按钮
	$(".qrcode").qrcode({
	render: "canvas",
	width: 166,
	height: 166,
	text: url
});
</pre>
			</li>
		</ul>

		<h3></h3>
	</body>
</html>