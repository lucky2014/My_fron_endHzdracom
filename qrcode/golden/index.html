<!DOCTYPE html>
<!-- saved from url=(0089)file:///C:/Users/Administrator.LQ-20160307MDZP/AppData/Local/Temp/MarkdownPadPreview.html -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>砸金蛋</title>

<style type="text/css">
/* GitHub stylesheet for MarkdownPad (http://markdownpad.com) */
/* Author: Nicolas Hery - http://nicolashery.com */
/* Version: b13fe65ca28d2e568c6ed5d7f06581183df8f2ff */
/* Source: https://github.com/nicolahery/markdownpad-github */

/* RESET
=============================================================================*/

html, body, div, span, applet, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, a, abbr, acronym, address, big, cite, code, del, dfn, em, img, ins, kbd, q, s, samp, small, strike, strong, sub, sup, tt, var, b, u, i, center, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, canvas, details, embed, figure, figcaption, footer, header, hgroup, menu, nav, output, ruby, section, summary, time, mark, audio, video {
  margin: 0;
  padding: 0;
  border: 0;
}

/* BODY
=============================================================================*/

body {
  font-family: Helvetica, arial, freesans, clean, sans-serif;
  font-size: 14px;
  line-height: 1.6;
  color: #333;
  background-color: #fff;
  padding: 20px;
  max-width: 960px;
  margin: 0 auto;
}

body>*:first-child {
  margin-top: 0 !important;
}

body>*:last-child {
  margin-bottom: 0 !important;
}

/* BLOCKS
=============================================================================*/

p, blockquote, ul, ol, dl, table, pre {
  margin: 15px 0;
}

/* HEADERS
=============================================================================*/

h1, h2, h3, h4, h5, h6 {
  margin: 20px 0 10px;
  padding: 0;
  font-weight: bold;
  -webkit-font-smoothing: antialiased;
}

h1 tt, h1 code, h2 tt, h2 code, h3 tt, h3 code, h4 tt, h4 code, h5 tt, h5 code, h6 tt, h6 code {
  font-size: inherit;
}

h1 {
  font-size: 28px;
  color: #000;
}

h2 {
  font-size: 24px;
  border-bottom: 1px solid #ccc;
  color: #000;
}

h3 {
  font-size: 18px;
}

h4 {
  font-size: 16px;
}

h5 {
  font-size: 14px;
}

h6 {
  color: #777;
  font-size: 14px;
}

body>h2:first-child, body>h1:first-child, body>h1:first-child+h2, body>h3:first-child, body>h4:first-child, body>h5:first-child, body>h6:first-child {
  margin-top: 0;
  padding-top: 0;
}

a:first-child h1, a:first-child h2, a:first-child h3, a:first-child h4, a:first-child h5, a:first-child h6 {
  margin-top: 0;
  padding-top: 0;
}

h1+p, h2+p, h3+p, h4+p, h5+p, h6+p {
  margin-top: 10px;
}

/* LINKS
=============================================================================*/

a {
  color: #4183C4;
  text-decoration: none;
}

a:hover {
  text-decoration: underline;
}

/* LISTS
=============================================================================*/

ul, ol {
  padding-left: 30px;
}

ul li > :first-child, 
ol li > :first-child, 
ul li ul:first-of-type, 
ol li ol:first-of-type, 
ul li ol:first-of-type, 
ol li ul:first-of-type {
  margin-top: 0px;
}

ul ul, ul ol, ol ol, ol ul {
  margin-bottom: 0;
}

dl {
  padding: 0;
}

dl dt {
  font-size: 14px;
  font-weight: bold;
  font-style: italic;
  padding: 0;
  margin: 15px 0 5px;
}

dl dt:first-child {
  padding: 0;
}

dl dt>:first-child {
  margin-top: 0px;
}

dl dt>:last-child {
  margin-bottom: 0px;
}

dl dd {
  margin: 0 0 15px;
  padding: 0 15px;
}

dl dd>:first-child {
  margin-top: 0px;
}

dl dd>:last-child {
  margin-bottom: 0px;
}

/* CODE
=============================================================================*/

pre, code, tt {
  font-size: 12px;
  font-family: Consolas, "Liberation Mono", Courier, monospace;
}

code, tt {
  margin: 0 0px;
  padding: 0px 0px;
  white-space: nowrap;
  border: 1px solid #eaeaea;
  background-color: #f8f8f8;
  border-radius: 3px;
}

pre>code {
  margin: 0;
  padding: 0;
  white-space: pre;
  border: none;
  background: transparent;
}

pre {
  background-color: #f8f8f8;
  border: 1px solid #ccc;
  font-size: 13px;
  line-height: 19px;
  overflow: auto;
  padding: 6px 10px;
  border-radius: 3px;
}

pre code, pre tt {
  background-color: transparent;
  border: none;
}

kbd {
    -moz-border-bottom-colors: none;
    -moz-border-left-colors: none;
    -moz-border-right-colors: none;
    -moz-border-top-colors: none;
    background-color: #DDDDDD;
    background-image: linear-gradient(#F1F1F1, #DDDDDD);
    background-repeat: repeat-x;
    border-color: #DDDDDD #CCCCCC #CCCCCC #DDDDDD;
    border-image: none;
    border-radius: 2px 2px 2px 2px;
    border-style: solid;
    border-width: 1px;
    font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
    line-height: 10px;
    padding: 1px 4px;
}

/* QUOTES
=============================================================================*/

blockquote {
  border-left: 4px solid #DDD;
  padding: 0 15px;
  color: #777;
}

blockquote>:first-child {
  margin-top: 0px;
}

blockquote>:last-child {
  margin-bottom: 0px;
}

/* HORIZONTAL RULES
=============================================================================*/

hr {
  clear: both;
  margin: 15px 0;
  height: 0px;
  overflow: hidden;
  border: none;
  background: transparent;
  border-bottom: 4px solid #ddd;
  padding: 0;
}

/* TABLES
=============================================================================*/

table th {
  font-weight: bold;
}

table th, table td {
  border: 1px solid #ccc;
  padding: 6px 13px;
}

table tr {
  border-top: 1px solid #ccc;
  background-color: #fff;
}

table tr:nth-child(2n) {
  background-color: #f8f8f8;
}

/* IMAGES
=============================================================================*/

img {
  max-width: 100%
}
</style>
<!--<base href="file:\\\C:\Users\Administrator.LQ-20160307MDZP\Desktop\">--><base href=".">
</head>
<body>
<h2>砸金蛋使用方法</h2>
<ol>
<li>
<p>HTML代码</p>
<pre>
  &lt;div class="box main_box"&gt;
       &lt;ul data-type="true"&gt;
            &lt;li class="img1"&gt;
              &lt;img src="images/egg.png" alt=""&gt;
            &lt;/li&gt;
            &lt;li class="img2"&gt;
              &lt;img src="images/egg.png" alt=""&gt;
            &lt;/li&gt;
            &lt;li class="img3"&gt;
              &lt;img src="images/egg.png" alt=""&gt;
            &lt;/li&gt;
            &lt;li class="img4"&gt;
              &lt;img src="images/egg.png" alt=""&gt;
            &lt;/li&gt;
      &lt;/ul&gt;
      &lt;span class="hammer"&gt;&lt;img src="images/hammer.png" alt=""/&gt;&lt;span&gt;
  &lt;/div&gt;
</pre>
</li>
<li>
<p>js代码</p>
<pre>
var options = {
      imgSelected:".img3",//选中的列表className名
      timer:300,//几秒后执行回调函数
      imgArr:["images/egg.png","images/eggBreak.png"],//传递的蛋图片
      callback:function(){//蛋裂开的毁掉函数
        alert("蛋砸碎了")
      }
};
$(".main_box").smashing(options);
</pre>
</li>
<li>
<p>样式</p>
<pre>
body{
  position: relative;
  max-width:768px;
  margin:0 auto;
  margin-top:100px;
}
ul,li{
  list-style: none;
}
.box{
  position: absolute;
  top:36px;
  width:100%;
  height:100%;
}
.box img{
  width:100%; 
}
.box>img{
  width:92%;
  margin-left:4%;
}
.box ul{
  position: absolute;
  top:0;
  width:320px;
  height:100%;
  left:50%;
  margin-left:-160px;
}
.box ul li{
  width:80px;
  transition:all 1s ease;
  position:absolute;
}
.box ul li.img1{
  transform: translate(120px,-80px);
  -webkit-transform: translate(120px,-80px);
}
.box ul li.img2{
  transform: translate(10px,-30px);
  -webkit-transform: translate(10px,-30px);
}
.box ul li.img3{
  transform: translate(120px,40px) scale(1.8);
  -webkit-transform: translate(120px,40px) scale(1.8);
  z-index:300;
}
.box ul li.img4{
  transform: translate(230px,-30px);
  -webkit-transform: translate(230px,-30px);
}
.box1{
  position: absolute;
  top:0;
}
.hammer{
  transform-origin:100% 100%;
  display: inline-block;
  width:90px;
  position: absolute;
  left:50%;
  transform: rotate(14deg);
  -webkit-transform: rotate(14deg);
  margin-left:7%;
  top:-20px;
  z-index:300;
}
@media screen and (max-width:480px){
  .hammer{
    left:55%;
  }
}
@media screen and (max-width:380px){
  .hammer{
    left:56%;
  }
}
@media screen and (max-width:340px){
  .hammer{
    left:58%;
  }
}
.hammer_hover{
  -webkit-animation:hammer .4s infinite alternate  ease;
  -moz-animation:hammer .4s infinite alternate  ease;
  -ms-animation:hammer .4s infinite alternate  ease;
  -o-animation:hammer .4s infinite alternate  ease;
  animation:hammer .4s infinite alternate  ease;
  -webkit-animation-iteration-count:1;
  -moz-animation-iteration-count:1;
  -ms-animation-iteration-count:1;
  -o-animation-iteration-count:1;
  animation-iteration-count:1;
}
.breaking{
  -webkit-animation:breaking .4s infinite alternate  ease;
  -moz-animation:breaking .4s infinite alternate  ease;
  -ms-animation:breaking .4s infinite alternate  ease;
  -o-animation:breaking .4s infinite alternate  ease;
  animation:breaking .4s infinite alternate  ease;
  -webkit-animation-iteration-count:1;
  -moz-animation-iteration-count:1;
  -ms-animation-iteration-count:1;
  -o-animation-iteration-count:1;
  animation-iteration-count:1;
}
.shake{
  transform-origin:50% 100%;
  -webkit-animation:shake .1s infinite alternate  ease;
  -moz-animation:shake .1s infinite alternate  ease;
  -ms-animation:shake .1s infinite alternate  ease;
  -o-animation:shake .1s infinite alternate  ease;
  animation:shake .1s infinite alternate  ease;
  -webkit-animation-iteration-count:3;
  -moz-animation-iteration-count:3;
  -ms-animation-iteration-count:3;
  -o-animation-iteration-count:3;
  animation-iteration-count:3;
}
@-webkit-keyframes hammer{
    0%{
        transform:rotate(90deg)
    }
    100%{
        transform: rotate(14deg);
    }
}

@-webkit-keyframes gift{
    0%{
        transform:translate(24px,-60px) scale(0)
    }
    100%{
        transform:translate(60px,-90px) scale(4);
    }
}
@-webkit-keyframes shake{
    0%{
        transform:rotate(-1deg)
    }
    100%{
        transform:rotate(1deg)
    }
}
@-webkit-keyframes breaking{
    0%{
      opacity: 0;
        transform:translateY(60px) scale(0);
    }
    100%{
      opacity: 1;
        transform:translateY(0) scale(1);
    }
}</pre>
</li>
<li>
<p><a href="http://122.224.96.245/index/golden/demo/index.html">demo</a></p>
</li>
<li>
<p><a href="http://122.224.96.245/index/golden/demo.rar">下载</a></p>
</li>
</ol>



<!-- This document was created with MarkdownPad, the Markdown editor for Windows (http://markdownpad.com) -->
</body></html>